pulsar-transaction/coordinator/src/main/java/org/apache/pulsar/transaction/coordinator/impl/MLTransactionMetadataStore.java

```diff
@@ -129,7 +130,6 @@ public void replayComplete() {
                     } else {
```

Comment:
better to move this line, because 
https://github.com/apache/pulsar/pull/18924/files#diff-70bdccda00334bdcc596cb6fea9f166021fc858af8c0432a3ee86a6911150d07R123
in some case the tc will not recover success